### DISCUSSION TITLE
Mandatory compliance webhooks

### DISCUSSION THREAD

**Discussion by TNagendra** [kinngh/shopify-node-express-mongodb-app#131] [Q&A] [answered]
Please help me.
How to trigger Mandatory compliance webhooks. Can you tell me this briefly ? 
when i am going to run this command line "shopify webhook trigger --topic=gdpr/topic --api-version=2024-01 --client-secret= --delivery-method=http --address=ngrokurl.com/api/gdpr/topic" 
I'm getting like "Topic 'gdpr/topic' does not exist for ApiVersion '2024-01'" this error in teriminal.

**Comment by kinngh** [comment, marked as answer]
It's because GDPR topics aren't part of the base webhook system. They're just POST routes, so in dev mode you can add in a conditional to skip the check or reverse and create your own HMACs and pass in the headers to your requests. To see how to create your own HMAC, read through the middleware and generate it.
